components/slider/index.css

```diff
@@ -10,63 +10,94 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-slider.css";
+
+
+/* missing tokens for variant with Tick, also need t-shirt sizing */
+.spectrum--medium {
+  --spectrum-slider-tick-mark-height: 10px;
+  /* --spectrum-global-dimension-size-125 */
+  --spectrum-slider-tick-mark-width: 2px;
+  /* --spectrum-global-dimension-size-25: */
+  --spectrum-slider-tick-mark-border-radius: 2px;
```

Comment:
`--spectrum-slider-tick-mark-border-radius: 2px;`

Might this be redundant since it is the same as the width?

Reply:
I think those should be still two different vars as we will replace those hard-coded values soon with tokens.